appconfig/settings/wdb.q

```diff
@@ -0,0 +1,5 @@
+// Bespoke WDB settings for AWS finspace Starter Pack
```

Comment:
Instead of this comment, can we have a comment that finspace wdb is designed to be 1:1 relationship periods? Every new period will have a new wdb